src/app/[locale]/(default)/events/loading.tsx

```diff
@@ -0,0 +1,27 @@
+import { EventCardSkeleton } from '@/components/events/EventCardSkeleton';
+import { getTranslations } from 'next-intl/server';
+
+export default async function EventsSkeleton() {
+  const t = await getTranslations('events');
+  return (
+    <>
+      <h1 className='my-4'>{t('title')}</h1>
+      <h2 className='my-2'>{t('activeEvents')}</h2>
```

Comment:
```suggestion
      <h2 className='my-4'>{t('activeEvents')}</h2>
```
Consistent styling with other h2 elements